app/config.py

```diff
@@ -32,6 +32,8 @@ class Scheduler(BaseModel):
     cleanup_client_directory_after_directory_delete: bool = Field(
         default=True,
     )
+    ignore_directory_after_success_timeout: str
+    ignore_directory_after_failed_attempts_threshold: int
```

Comment:
similar to other time-based config properties, best for these values to have a timeout
```suggestion
    ignore_directory_after_success_timeout: str = Field(default=120s)
    ignore_directory_after_failed_attempts_threshold: int = Field(default=5)
```